gradle.properties

```diff
@@ -1 +1 @@
-version=0.1.0-SNAPSHOT
\ No newline at end of file
+version=0.1.0-candidate-1
```

Comment:
the version should be 0.1.0 . The RC becomes the released version 0.1.0 if the vote passes.